### ISSUE TITLE
UI field under TABs field cause "TypeError: addSubField is not a function"

### ISSUE THREAD

**Issue by khakimvinh** [payloadcms/payload#898]
# Bug Report

When using UI field under tabs field, payload hit error `TypeError: addSubField is not a function`, the same UI field work fine if put directly under collection.

## Expected Behavior

Able to use ui field under tabs without error

## Current Behavior

Error `TypeError: addSubField is not a function` when try to run dev

## Possible Solution

Unknown

## Steps to Reproduce

1. Clone public-demo repo
2. Create SampleComponent: 
```ts
const SampleComponent: React.FC = () => <div>Sample</div>;
```
3. Add a tabs field, add ui field, add `SampleComponent` into any one collection:
```js
{
      type: 'tabs',
      tabs: [
        {
          label: 'Tab 1',
          fields: [
            {
              type: 'ui',
              name: 'field1',
              label: 'Field 1',
              admin: {
                components: {
                  Field: SampleComponent,
                },
              },
            },
          ],
        },
      ],
    }
```
4. Run payload 
  ```bash
    yarn install && yarn dev
  ```

## Detailed Description
- Payload version: `1.0.16`
- Error message: 
```bash
TypeError: addSubField is not a function
    at /home/projects/public-demo/node_modules/payload/src/graphql/schema/buildMutationInputType.ts:192:18
    at Array.reduce (<anonymous>)
    at /home/projects/public-demo/node_modules/payload/src/graphql/schema/buildMutationInputType.ts:190:23
    at Array.reduce (<anonymous>)
    at tabs (/home/projects/public-demo/node_modules/payload/src/graphql/schema/buildMutationInputType.ts:187:90)
    at /home/projects/public-demo/node_modules/payload/src/graphql/schema/buildMutationInputType.ts:211:12
    at Array.reduce (<anonymous>)
    at buildMutationInputType (/home/projects/public-demo/node_modules/payload/src/graphql/schema/buildMutationInputType.ts:202:20)
    at /home/projects/public-demo/node_modules/payload/src/collections/graphql/init.ts:131:85
    at Array.forEach (<anonymous>)
```

**Comment by jmikrut** [comment]
Nice and easy one to fix. Thanks for finding @khakimvinh — fix incoming shortly!

**Comment by jmikrut** [comment]
Fixed in `1.0.17`. Thank you!
